app/cdap/components/WranglerGrid/SelectColumnPanel/CountWidget/index.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
```

Comment:
this component might not be needed as a separate one as we are using it in single place
if otherwise, plz provide more information in the form of one liner comments in code and here in the review comments as well
plz make sure we don't keep unwanted files and make the components compact

Reply:
This Component is reusable, We need it in add transformation panel feature also.